packages/blocks/src/store/utils.js

```diff
@@ -18,3 +18,15 @@ export const getValueFromObjectPath = ( object, path, defaultValue ) => {
 	} );
 	return value ?? defaultValue;
 };
+
```

Comment:
Note to self: Add JSDoc.

Reply:
https://github.com/WordPress/gutenberg/pull/62272/commits/6dd85a715783050483295f2c7fe133dcb1230442